buildkite/pipelines/gerrit-postsubmit.yml

```diff
@@ -7,6 +7,18 @@ platforms:
     - "--test_tag_filters=-slow"
     test_targets:
     - "..."
+  ubuntu1804_java9:
```

Comment:
On Bazel@HEAD, JDK10 javac upgrade was reverted, does it mean we would have to wait for JDK10 support? Something like that:

```
  bazel build --host_java_toolchain=@bazel_tools//tools/jdk:toolchain_java10 \
    --java_toolchain=@bazel_tools//tools/jdk:toolchain_java10
```

But please, don't add this yet, because the change that supports that on Gerit side wasn't merged yet: [1].

* [1] https://gerrit-review.googlesource.com/c/gerrit/+/193832

Reply:
Added a commented-out definition and a TODO.